src/extra/Markdown/plugins/decorator.ts

```diff
@@ -90,9 +102,12 @@ export const decoratorPlugin: unified.Plugin<[DecoratorPluginOptions]> = (
 		return;
 	}
 
-	const visitor: visitParents.Visitor<any> = (node, parents) => {
+	const visitor: visitParents.Visitor<Node> = (node, parents) => {
+		if (!isLiteralNode(node) || typeof node.value !== 'string') {
+			return;
+		}
 		const decorator = options.decorators!.find((entry) => {
-			return node.type === 'text' && entry.match.test(node.value);
+			return typeof node.value === 'string' && entry.match.test(node.value);
```

Comment:
`typeof node.value === 'string'` is already written above.

Reply:
Nice spot :+1: